packages/vc.js/package.json

```diff
@@ -50,6 +50,7 @@
     "typescript": "^3.9.5"
   },
   "dependencies": {
+    "@transmute/did-key-ed25519": "^0.3.0-unstable.5",
```

Comment:
hmm seems like a mistake to put this here.

Reply:
I will see if we can remove this!

Reply:
This has been removed as a depency. is currently a dev dependency for tests.